src/utils/signer.ts

```diff
@@ -72,6 +73,12 @@ export class Signer {
             signature,
             protocolFee,
         );
+        logger.info({
+            message: `attempting to sign and broadcast a meta transaction`,
+            nonce: ethereumTxnParams.nonce,
```

Comment:
Would be better to convert these from hex values otherwise we'll have to manually do it when looking at the logs